wordle-web/src/views/AboutView.vue

```diff
@@ -0,0 +1,23 @@
+<template>
```

Comment:
This page meets all the technical requirements, but they wanted it to be named game.vue

Reply:
Thank you, I changed the name to game.vue.